src/contacts/tests/test_db.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tests for db module
+"""
+import random
+
+import unittest
+
+from contacts.db import ContactsDb
+from contacts.tests.constants import EXAMPLE_CONTACT_DB_OBJ
+
+
+class TestDb(unittest.TestCase):
+    """
+    Test cases for db module
+    """
+
+    def setUp(self):
+        """Init db and create table before each test"""
+        # init SQLAlchemy with sqllite in mem
+        self.db = ContactsDb("sqlite:///:memory:")
+        # create contacts table in mem
+        self.db.contacts_table.create(self.db.engine)
+
+    def test_add_contact_returns_none_no_exception(self):
+        """test if a contact can be added"""
```

Comment:
i wonder if there is a way to explicitly test that no exception was raised

Reply:
From my understanding, the expectation of unittest is that it will fail for _any_ exception and hence there is no need for an explicit assert.